xline-client/tests/maintenance.rs

```diff
@@ -0,0 +1,27 @@
+use xline_client::{clients::maintenance::MaintenanceClient, error::Result, Client, ClientOptions};
+use xline_test_utils::Cluster;
+
+#[tokio::test]
+async fn snapshot() -> Result<()> {
+    let (_cluster, mut client) = get_cluster_client().await?;
+
+    let mut msg = client.snapshot().await?;
+    loop {
+        if let Some(resp) = msg.message().await? {
+            assert!(!resp.blob.is_empty());
+            if resp.remaining_bytes == 0 {
+                break;
+            }
+        }
+    }
+    Ok(())
+}
+
+async fn get_cluster_client() -> Result<(Cluster, MaintenanceClient)> {
```

Comment:
There is not need to return Cluster in `get_cluster_client`

Reply:
I think we need to return it, because the cluster will stop when it is dropped.